chain/chunks/README.md

```diff
@@ -0,0 +1,30 @@
+# near-chunks
+
+This crate cotains functions to handle chunks. In NEAR - the block consists of multiple chunks - at most one per shard.
+
+When chunk is created, the creator encodes it using Reed Solomon encoding (ErasureCoding) - creating PartialChunks, that are later sent to all the validators (each validator gets a subset of them).
+
+Each validator is waiting to receive its share of PartialChunks (`has_all_parts`) - and once they are verified, it can accept the chunk (and if all chunks are accepted chain/client can sign the block).
```

Comment:
I think it might be nice to differentiate between nodes and validators here because this algorithm/code is used for both non-validators and validators. For example, we could mention,

"A honest validator will only approve a block if it receives its assigned parts for all chunks in the block"
"For all nodes (validators and non-validators), they will only accept/process a block if the following requirements are satisfied: 
- for every shard it tracks, a node has to have the full chunk,
- for every shard it doesn't track, a node has have the receipts from this shard to all shards
"
Feel free to reword and choose the level of details you want to include in this README

Reply:
ack